samples/highcharts/data-tools/datacursor-group/demo.css

```diff
@@ -0,0 +1,38 @@
+#wrapper {
+    display: flex;
+    justify-content: stretch;
```

Comment:
Can we please add for example `gap: 20px`?
At first I though it is one big table ;)